README.md

```diff
@@ -53,6 +54,10 @@ targets:
 
 Tasks can also be executed via puppet by specifying `user`, `password`, and `instance` parameters. Credentials specified via parameters will take precedence over any in an inventory file.
 
+## OAuth
+
+The user can also authenticate using OAuth2 by setting the `access_token` parameter on any of the tasks. This will be prioritized over the user/password specified in your inventory file.
```

Comment:
Hmm, I think we should make this more generic. So, just talk about how to specify credentials, how that reduces to either oauth token or username/password, how those can be specified in inventory file (with examples) or as task params to override what's in inventory.